projects/uitdatabank/docs/search-api/advanced/advanced-queries.md

```diff
@@ -958,6 +958,45 @@ Retrieve all events and places that have 5 to 10 videos:
 GET /offers/?q=videosCount:[5 TO 10]
 ```
 
+### completeness
+
+With the `completeness` field you can filter events, places and organizers by their completeness score. The completeness score is a number between 0 and 100 that indicates how complete the information of an event, place or organizer is.
```

Comment:
Completely out of scope and probably too confusing for users of the documentation, but I just realized that since certain fields are mandatory, the completeness score can never be lower than 36/48

Reply:
I think we still have some invalid JSON projections that have a lower score. A replay should fix those.